### ISSUE TITLE
`Assimilasjon` og `Assimilering`

### ISSUE THREAD

**Issue by danmichaelo** [realfagstermer/realfagstermer#206]
Brukt i litt ulike betydninger, vi bør få på plass en definisjon

http://ask.bibsys.no/ask/action/result?cql=%28bs.lokoeo-frase+%3D+%22Assimilasjon%22%29++AND+%28bs.avdeling+%3D+%22ureal?%22%29&aktivKilde=biblio%3AUBO

http://ask.bibsys.no/ask/action/result?cmd=&kilde=biblio&cql=%28bs.lokoeo-frase+%3D+%22Assimilering%22%29+AND+%28bs.avdeling+%3D+%22ureal%3F%22%29

**Comment by bibliomari** [comment]
@BioHeidi @Hypsibius Vi har lagt inn Assimilering under Assimilasjon, men ser at vi også har emneordet Nitrogenassimilasjon. Hvordan forholder disse to seg til hverandre? Vi rydder bort de evt 2 bøkene som ikke handler om planter.

**Comment by mittinatten** [comment]
Da regner jeg med termen skal slettes fra denne http://ask.bibsys.no/ask/action/show?kid=biblio&cmd=reload&pid=021911223 
Vet ikke helt hva "Data assimilation" betyr og om vi trenger en term for det. (@mzyg?) Er det det samme som `Dataintegrasjon`?

**Comment by bibliomari** [comment]
JA, jeg ønsker meg selvsagt Assimilasjon (meteorologi) på den boka @mittinatten  :dancers:

**Comment by mzyg** [comment]
Vi burde ha 'dataassimilasjon EN data assimilation' som en egen term (assimilasjon av observeringsdata i klimamodeller /værmodeller ) 
http://no.wikipedia.org/wiki/Dataassimilasjon
Dataintegrasjon er feil her.

**Comment by BioHeidi** [comment]
Ser at vi i hovedsak har brukt assimilasjon i betydningen nitrogenassimilasjon. Nitrogenassimilasjon kan godt stå inntil videre, mulig den kan ryddes mot nitrogenfiksering og nitrifikasjon. Men det trenger altså ikke å stå assimilasjon i tillegg.
